cmd/ptp4u/main.go

```diff
@@ -117,6 +119,7 @@ func main() {
 	s := server.Server{
 		Config: c,
 		Stats:  st,
+		Check:  &drain.FileDrain{File: "/var/tmp/kill_ptp4u"},
```

Comment:
Can we define several checks? File + something else?

Reply:
Which other check do you have in mind?

Reply:
Host status in some asset database :)